sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpLanguageConfigurationTest.java

```diff
@@ -37,4 +39,34 @@ public void reads_correct_language() {
 
     assertThat(config.bugCategories()).containsExactly("C#");
   }
+
+  @Test
+  public void whenSettingIsTrue_analyzeRazorCode_returnsTrue() {
```

Comment:
Too bad we are using junit 5 only in ITs. We could have used parameterized tests.